### PR TITLE
Upgrade installation instructions

### PR DESCRIPTION
**Pull request by ttaiv**
- Add explicit step for installing FreeSurfer.
- Mention creation of virtual environment (uv/conda/venv).
- Guide to install exact PyTorch version for the hardware.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,69 +21,66 @@ For more information about the method, please see:
...
+   git clone https://github.com/johnsam7/ceremegbellum.git
+   cd ceremegbellum
    ```
 
-2. **Install compiled dependencies as pre-built wheels:**
+3. Create a new virtual environment for Cere-MEG-Bellum and activate it.
+You can do this, for example, with [uv](https://docs.astral.sh/uv/) (recommended!),
+[venv](https://docs.python.org/3/library/venv.html) or [conda](https://github.com/conda/conda).
+`uv` and `conda` are good options because they can also manage the Python version without
+depending on system Python.
```
[the remark this answers is not in the document]
Comment (ttaiv) [reply README.md]: Suggestion implemented.

File: README.md [hunk 2]
```diff
@@ -21,69 +21,66 @@ For more information about the method, please see:
...
-   pip install --only-binary :all: \
-       numpy scipy pandas matplotlib torch antspyx \
-       scikit-image scikit-learn imagecodecs numexpr blosc2 \
-       Pillow connected-components-3d SimpleITK \
-       "timm<1.0.23" torchvision einops seaborn dicom2nifti
+   # uv
+   uv venv cmb-env --python 3.12 --seed
+   source cmb-env/bin/activate
+   # conda
+   conda conda create -n cmb-env python=3.12 pip
```
[the remark this answers is not in the document]
Comment (ttaiv) [reply README.md]: Fixed.

File: README.md [hunk 3]
```diff
@@ -21,69 +21,66 @@ For more information about the method, please see:
...
+   conda activate cmb-env
+   # venv
+   python -m venv cmb-env
+   source cmb-env/bin/activate
    ```
 
-3. **Install source-only packages (pure Python, no compiler needed):**
+4. Install correct PyTorch version for your hardware. Look up the exact installation on [PyTorch webpage](https://pytorch.org/get-started/locally/).
+   Here is **example** command for hardware with NVIDIA GPU with CUDA 12.6 support.
```
[the remark this answers is not in the document]
Comment (ttaiv) [reply README.md]: Wording adjusted.
